src/components/AppLoading.tsx

```diff
@@ -25,6 +28,41 @@ export const AppLoading: FC<PropsWithChildren> = ({ children }) => {
 
   const isLoading = !isLoadingComplete || isSignedIn === null || !isLayoutReady
 
+  const currentVersion = Application.nativeApplicationVersion || 'unknown'
+  const os = Platform.OS
```

Comment:
destructuring, up to you

```suggestion
  const { OS } = Platform
```

Reply:
I've tried but the swagger's schema includes os so while destructuring, is not working properly 

![Captura de pantalla 20 05 2024 a 08 49 24 a  m](https://github.com/binarapps/baca-react-native-template/assets/140461741/310fafc2-e0a6-4a3a-a613-3ea1c80cc6a4)

Reply:
In this case in data at the end point, can be: 
{
currentVersion,
os:OS
},
as I said, up to you